### ISSUE TITLE
Automatically update gh-pages after Documentation and examples have been committed

### ISSUE THREAD

**Issue by megoth** [solid/solid-ui#271]
Currently there seems to be a lack of automatic update of gh-pages when new documentation is added. We also need to generate the API documentation using `npm run doc` for gh-pages, since I think it's bad practice to commit files that are automatically generated anyway.

**Comment by michielbdejong** [comment]
Yes. I would say  it's hard to determine if any typedoc comment blocks or examples pages changed, so just tell Trabvis tfor every master branch commit to:
* check out gh-pages
* merge in the master branch
* npm run build (so that lib/webpack-bundle.js is generated, which is needed for the examples)
* npm run doc ( so that Documentation/api. is generated)

Currently, we have been merging the master branch into the gh-pages branch manually, and then running `npm run build` and `npm run doc` on it.
https://solid.github.io/solid-ui/examples/buttons/ works
https://solid.github.io/solid-ui/Documentation/form-ecosystem.html works
https://solid.github.io/solid-ui/Documentation/api/index.html doesn't, which is weird.
